src/test/groovy/com/palantir/gradle/javadist/JavaDistributionPluginTests.groovy

```diff
@@ -309,6 +310,27 @@ class JavaDistributionPluginTests extends GradleTestSpec {
         expectedCheckConfig == actualCheckConfig
     }
 
+    def 'produce distribution bundle with custom classpath'() {
+        given:
+        createUntarBuildFile(buildFile)
+        buildFile << '''
+            dependencies { compile files("external.jar") }
+            tasks.jar.baseName = "internal"
+            distribution {
+                customClasspath 'manager.jar', 'config.jar'
+            }
+        '''.stripIndent()
+        file('src/main/java/test/Test.java') << "package test;\npublic class Test {}"
+
+        when:
+        runSuccessfully(':build', ':distTar', ':untar')
```

Comment:
Good point. I changed it to require that the customClasspath jars have to match actual dependency jars.